src/TypeAutomata/Determinize.hs

```diff
@@ -80,19 +80,24 @@ combineNodeLabels (fstLabel@MkNodeLabel{}:rs) =
   case rs_merged of
     pr@MkPrimitiveNodeLabel{} -> error ("Tried to combine primitive type" <> show pr <> " and algebraic type " <> show fstLabel)
     combLabel@MkNodeLabel{} ->
```

Comment:
Es gibt in dieser Funktion keinen Test der auf Gleichheit von Kinds testet? (Zusätzlich zu dem Test der Gleichheit von Polarity).